cocos/editor-support/cocostudio/ActionTimeline/CCBoneNode.h

```diff
@@ -0,0 +1,197 @@
+/****************************************************************************
+Copyright (c) 2014 cocos2d-x.org
+
+http://www.cocos2d-x.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+****************************************************************************/
+
+
+#ifndef __CCBONENODE_H__
+#define __CCBONENODE_H__
+
+#include "base/CCProtocols.h"
+#include "2d/CCLayer.h"
+#include "CCTimelineMacro.h"
+#include "cocostudio/CocosStudioExport.h"
+
+#include "CCSkinNode.h"
+
```

Comment:
can not use forward declaration for class SkinNode in namespace timeline, because SkinNode is a typedef of Node,   forward declare would make a redefine error